include/deal.II/grid/tria.h

```diff
@@ -939,7 +939,6 @@ namespace internal
  *       std::cout << "Triangulation has been refined." << std::endl;
  *     }
  *
- *     void f() {
```

Comment:
Ah, I see. The example is intended to use the first `f()` as a slot for the signal. It is unfortunate that I named the second function `f()` as well. Let's rename the second function `main()`.